oioioi/mp/templates/mp/ranking.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n pagination_tags get_user_name static simple_filters %}
```

Comment:
This file looks like 1:1 to `oioioi/oioioi/rankings/templates/rankings/default_ranking.html `. If it is unmodified I think the code should be restructured not to repeat ourself's